README.md

```diff
@@ -146,17 +161,66 @@ Note that you'll need to pass the configuration file each time you run `mix` wit
 
 **NOTE** If you're using persistent `geth` and `geth` is restarted after the above step, the authority account must be unlocked again:
 
-```
+```bash
 geth attach http://127.0.0.1:8545
 personal.unlockAccount(“<authority_addr from ~/config.exs>”, 'ThisIsATestnetPassphrase', 0)
 ```
 The passphrase mentioned above originates from [`dev_helpers`](apps/omg_eth/test/support/dev_helpers.ex).
 It is what is used when deploying the contract in the `dev` environment using `prepare_env!()` as above.
 
+##### Manual contract deployment
+
+Alternatively, the contract can be deployed manually. This is especially helpful when running on a `geth` instance 
+with limited `Ether` supply. From `elixir-omg` root directory:
+```bash
+mix compile
+iex -S mix run --no-start
+``` 
+       
+And then in `IEx`
+```elixir
+alias OMG.Eth.DevHelpers
+alias OMG.Eth
+
+import Eth.Encoding
+
+{:ok, _} = Application.ensure_all_started(:ethereumex)
+
+{:ok, accounts} = Ethereumex.HttpClient.eth_accounts()
+faucet = accounts
+         |> Enum.at(0) # assuming that this is an unlocked faucet
+
+# create and unlock authority account
+password = "secret_password"
+{:ok, authority} = Ethereumex.HttpClient.request(:personal_newAccount, ["password"], [])
+{:ok, true} = Ethereumex.HttpClient.request(:personal_unlockAccount, [authority, "password", 0], [])
+
+authority_raw = from_hex(authority)
+
+# fund the authority
+value = :math.pow(10, 18)
+        |> round
+        |> to_hex()
+{:ok, _} = Ethereumex.HttpClient.eth_send_transaction(%{from: faucet, to: authority, value: value})
+
+#deploy the contract
+{:ok, _} = deploy_result = Eth.RootChain.create_new("./", authority_raw)
```

Comment:
instead of this assignment, you should just `...create_new(...) |> deploy_sync!()`, _et voila_